src/main/scala/com/cognite/sdk/scala/v1/fdm/views/ViewPropertyCreateDefinition.scala

```diff
@@ -22,23 +25,15 @@ object ViewPropertyCreateDefinition {
   implicit val createViewPropertyDecoder: Decoder[CreateViewProperty] =
     deriveDecoder[CreateViewProperty]
 
-  final case class ConnectionDefinition(
```

Comment:
I think maybe what needs to happen here is that maybe it should be `CreateConnectionDefinition` and kept here :thinking: will need to recheck of Create* classes difference with others

Reply:
Yeah it's an option to duplicate the entire connectiondefinition/edgeConnection/RDRConnection into both create and read properties.

In this case I though it made sense to not duplicate them because in the specs, they use the same objects for all of them: 
![image](https://github.com/cognitedata/cognite-sdk-scala/assets/167076615/ca9ffb61-c571-4d8e-9f98-55f7640bd6b0)
![image](https://github.com/cognitedata/cognite-sdk-scala/assets/167076615/fd472d35-53ac-478b-8a6d-2fadb16c66c1)
As you can see in these two screenshot, the CreateViewProperty and ViexCorePropertyDefinitions are different, but the ConnectionDefinition are supposed to be the same object

Reply:
As of now, if we had connection definition both in create and in read, the entire code would be a duplicate for both the EdgeConnection and the ReverseDirectRelationConnection case classes. This doesn't mean it is totally wrong to do it like that, but I think since the spec define them as the same objects it make sense to reuse them.

Reply:
Hmm, yeah, will need to check if we can keep the seals and still have some reuse

Reply:
I think that's not possible unless you put both classes in the same file, but I might be wrong